pkg/odo/genericclioptions/context.go

```diff
@@ -387,6 +387,15 @@ func resolveApp(command *cobra.Command, createAppIfNeeded bool, localConfigurati
 	return app
 }
 
+// ResolveAppFlag resolves the app from the flag
+func ResolveAppFlag(command *cobra.Command) string {
```

Comment:
Can this func call `resolveApp()`, seems like we're doing the same thing except get from local config?

Reply:
I think `resolveApp` does more then this function and thats the reason I created this.